docs/topics/modules/glossary.adoc

```diff
@@ -16,3 +16,15 @@ Pipeline:: Pipelines are a continuous delivery system that, at each step, tests
 Scenario-Driven Planning:: A software development methodology focused on real-world problems, or Scenarios, described in the language and from the viewpoint of the user. Scenarios deliver particular Value Propositions and are realized through Experiences.
 
 Work Item:: Work Items describe and keep track of work that needs to be done. A user can be assigned to a Work Item. Each Work Item must be attached to a Collaboration Space and an Area (assigned by default). This can be used to model bugs, tasks, features, ideas, and more.
+
+Usage Outliers:: Components of your software stack that are not commonly used in other similar open source stacks or that rarely go together.
+
+Security Issues:: {osio} analyzes the CVEs of all your components and flags components that have security vulnerabilities.
+
+License outliers:: Licenses which rarely go together or not commonly used in other similar stacks.
+
+Alternate Components:: Components recommended by {osio}, to replace License or Usage Outliers.
```

Comment:
Alternate components are recommended in case of Usage and License Outliers in the Stack Report and in Che in the case of Security issues, hence listed it below those three components. Also this follows the UX order of filters in the detailed analysis of the table. Are you saying this cause it is a recommendation and the rest are observations based on the data?

Reply:
> Are you suggesting this because it is a recommendation and the rest are observations based on the data?

Yes.